src/plugins/discover/public/components/discover_container/discover_container.tsx

```diff
@@ -82,15 +82,13 @@ export const DiscoverContainerInternal = ({
       css={discoverContainerWrapperCss}
       data-test-subj="discover-container-internal-wrapper"
     >
-      <EuiFlexItem>
-        <KibanaContextProvider services={services}>
-          <DiscoverMainRoute
-            customizationCallbacks={customizationCallbacks}
-            mode="embedded"
-            isDev={isDev}
-          />
-        </KibanaContextProvider>
-      </EuiFlexItem>
+      <KibanaContextProvider services={services}>
+        <DiscoverMainRoute
+          customizationCallbacks={customizationCallbacks}
+          mode="embedded"
+          isDev={isDev}
+        />
+      </KibanaContextProvider>
```

Comment:
```suggestion
      <EuiFlexItem
        css={css`
          width: 100%;
        `}
      >
        <KibanaContextProvider services={services}>
          <DiscoverMainRoute
            customizationCallbacks={customizationCallbacks}
            mode="embedded"
            isDev={isDev}
          />
        </KibanaContextProvider>
      </EuiFlexItem>
```
Nit: the current fix works, but in general EUI recommends against using elements other than `EuiFlexItem` as direct descendants of `EuiFlexGroup`. I don't think it has any practical implications here, but just making the `EuiFlexItem` full width instead seems to fix the issue without breaking EUI recommendations.